code/index.html

```diff
@@ -3,62 +3,101 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
+    <title>GuessWho?</title>
     <link rel="stylesheet" href="style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet"
     />
+    <link href="https://fonts.googleapis.com/css2?family=Bangers&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Boogaloo&display=swap" rel="stylesheet">
+  </head>
   </head>
   <body>
     <aside class="question-section">
-      <button type="button" id="restart" class="outlined-button">RESTART</button>
+      
 
       <img
         class="guess-who-icon"
-        src="images/guess-who-bubble.png"
+        src="images/Guesswho_bubble.png"
         alt="Guess Who"
       />
 
-      <h1>Does the person have</h1>
+      <h1>Does the person have : </h1>
+
+       <!---- Drop down selection ---->
 
       <select title="questions" id="questions">
+        <option value="" selected disabled>Select attribute</option>
```

Comment:
Good choice to add this "non-option" as the first message shown before any of the options are selected.